executor/executor_txn_test.go

```diff
@@ -110,13 +110,13 @@ func TestInvalidReadTemporaryTable(t *testing.T) {
 	tk.MustExec("commit")
 
 	for _, query := range queries {
-		tk.MustExec(query.sql)
+		tk.MustQuery(query.sql)
```

Comment:
This change seems changing semantic. I'm curious how this change hardens the test?

Reply:
The result is not checked... I'm not modifing the logic here.

Reply:
Fine. It can be `MustExec` though.